keps/sig-node/127-user-namespaces/README.md

```diff
@@ -422,6 +422,31 @@ components that implement the interface.
 
 [kubeletVolumeHost-interface]: https://github.com/kubernetes/kubernetes/blob/36450ee422d57d53a3edaf960f86b356578fe996/pkg/volume/plugins.go#L322
 
+### Pod Security Standards (PSS) integration
+
+[Pod Security Standards](https://k8s.io/docs/concepts/security/pod-security-standards)
+define three different policies to broadly cover the whole security spectrum of
+Kubernetes, while the User Namespaces feature should integrate into them. This
+will happen only if the feature is graduated to GA, which _may_ result in
+changing the `Restricted` profile to disallow host user namespaces.
+
+With graduating the feature to beta, the Pod Security will relax in a controlled
+way for pods which enable user namespaces. This behavior can controlled by an API
+Server Feature Gate, which allows an early opt-in for end users. The overall
+burden to ensure that all nodes will honor user namespaces is on the cluster
+admin, though. The relaxation in detail means, that if user namespaces are
+enabled, then the following fields won't be restricted any more because they
+always have to refer to the user inside the container:
```

Comment:
I think the same reasoning also applies to allowPrivilegeEscalation, so it should be in this list.

Reply:
@destijl can you elaborate on that? How does `allowPrivilegeEscalation` (aka [`no_new_privs`](https://www.kernel.org/doc/Documentation/userspace-api/no_new_privs.rst)) interfere with user namespaces?

Reply:
I might be missing something subtle about user namespaces and the interaction with allowprivesc. But I read this list as "it doesn't make sense to have policy require you to set a non root user when using user namespaces, because letting you run as root is the point".

If that's the argument, I figured it also applied to allowprivesc. We don't care if you can privesc in your container if you're in a user namespace.